UI/src/components/molecules/RipaStopReason.vue

```diff
@@ -157,6 +157,42 @@
             :rules="explanationRules"
             @blur="handlePiiCheck($event)"
           ></ripa-text-input>
+          <template v-if="model.stopType === 'Vehicular'">
+            <ripa-form-header
+              title="The stopped person is a passenger in a vehicle"
+              v-on="$listeners"
+            >
+            </ripa-form-header>
+
+            <ripa-switch
+              v-model="model.person.passengerInVehicle"
+              label="The stopped person is a passenger in a vehicle"
+            ></ripa-switch>
+
+            <v-container>
+              <v-row no-gutters>
+                <v-col cols="12" sm="12"> </v-col>
+              </v-row>
+            </v-container>
+          </template>
+          <template v-if="model.stopType === 'Pedestrian'">
```

Comment:
I don't know the regulation text so this is a question, if the stop is a bicycle are either of these options supposed to be available?

Reply:
In my understanding the questions should not be available for a stop type bicycle. Only Vehicular or Pedestrian.